aries-site/src/examples/templates/dashboards/components/StatusBar.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import { Card, CardBody, CardHeader, Menu, NameValueList, Text } from 'grommet';
+import { MoreVertical } from 'grommet-icons';
+
+export const StatusBar = ({ children, title, ...rest }) => (
+  <Card {...rest}>
```

Comment:
The scaling here seems a little off -- the "unknown" bit is getting cut off. I would expect it to wrap or fit within the container.

<img width="1397" alt="Screen Shot 2022-02-02 at 10 56 32 AM" src="https://user-images.githubusercontent.com/12522275/152220741-19378256-e7e6-4152-87c1-4c832892d9f6.png">

Reply:
The purpose of this PR is to get some reusable components in place so that the rest of the Two-Column Dashboard work can be unblocked and distributed among us. Not focussed on those aspects at this time.